packages/belos/tpetra/src/solvers/Belos_Tpetra_GmresSstep.hpp

```diff
@@ -83,7 +84,8 @@ class CholQR {
     {
       auto R_h = R_mv.getLocalViewHost ();
       int ldr = int (R_h.extent (0));
-      lapack.POTRF ('U', ncols, R_h.data (), ldr, &info);
+      IST *Rimpl = reinterpret_cast<IST*> (R_h.data ());
```

Comment:
That's right -- that's how you do it : - ) .

Reply:
No.. It still did not work. I guess my IST is still Kokkos::complex<double>. So, I get "no known conversion to std::complex<double>". :(